pkg/drivers/qemu/qemu.go

```diff
@@ -408,6 +409,10 @@ func (d *Driver) Start() error {
 		startCmd = append(startCmd,
 			"-nic", fmt.Sprintf("user,model=virtio,hostfwd=tcp::%d-:22,hostfwd=tcp::%d-:2376,hostname=%s", d.SSHPort, d.EnginePort, d.GetMachineName()),
 		)
+	case "socket":
+		startCmd = append(startCmd,
+			"-nic", fmt.Sprintf("socket,model=virtio,sock=%s", d.NetworkSocket),
```

Comment:
Hi @AkihiroSuda, in your experience, do these parameters for the `-nic` flag look correct for socket_vmnet with minikube socket network?

Reply:
Looks good

Reply:
> `d.Network` is hardcoded so it's currently impossible to use `socket`.
> 
> When I do manually change `d.Network` to `socket` it fails because of line: `startCmd = append(startCmd, "-nic", fmt.Sprintf("socket,model=virtio,sock=%s", d.NetworkSocket)`
> 
> `d.NetworkSocket` isn't set anywhere in minikube, so it fails with: `ERROR: qemu-system-aarch64: -nic socket,model=virtio,sock=: Invalid parameter 'sock'`

Appreciate the comment, I have updated the code to reflect the network flag that is being passed into minikube rather than being hardcoded.

For now, if the network flag is empty, will default to the "user" network to maintain current functionality.

Once we have working flags for the qemu socket_vmnet network, will update the default network to "socket" for qemu.

Reply:
Hi @AkihiroSuda, we found some errors with these flags. I'll omit them for now but we have a follow-up PR that should more closely express working flags for minikube + qemu + socket_vmnet, appreciate your patience and review.